sample-skill/lambda/locale/en-US/LC_MESSAGES/data.po

```diff
@@ -7,7 +7,7 @@ msgid ""
 msgstr ""
```

Comment:
I would remove this files from the PR. It causes unnecessary noise. We can make a PR that updates all the translation files later.